File: Trial/update.php
<!DOCTYPE html>
<html>
<head>
	<title>Delete Book</title>
</head>
<body>
	<center>
	<?php
	
	if($_SERVER["REQUEST_METHOD"] == "POST"){
		$conn = mysqli_connect("localhost", "root", "", "book");
		
		// Check connection
		if(!$conn){
			echo " Database Error".mysqli_connect_error();
		}
		
		// Taking all 5 values from the form data(input)
		$id = $_REQUEST['id'];
				
		
		$sql = "SELECT * FROM book WHERE bookid='$id'";
      $result = mysqli_query($conn, $sql);

if (mysqli_num_rows($result) > 0) {
  // output data of each row
  while($row = mysqli_fetch_assoc($result)) {
    $id=$row["Bookid"];
	$name=$row["Name"];
	$author=$row["Author"];
	$publisher=$row["Publisher"];
	$price=$row["Price"];
	
  }
} else {
  echo "No book found";
}
	}
?>
	
<form method="post">
BookID:<input type="text" value="<?php echo $id;?>" name="bid"><br>
Book Name:<input type="text" value="<?php echo $name;?>" name="bname"><br>
Book Author:<input type="text" value="<?php echo $author;?>" name="bauth"><br>
Book Publisher:<input type="text" value="<?php echo $publisher;?>" name="bpub"><br>
Book price:<input type="text" value="<?php echo $price;?>" name="bprice"><br>
<input type="submit" value="update" name="update">
<?php
if(isset($_POST["update"]))
{
	$conn = mysqli_connect("localhost", "root", "", "book");
		
		// Check connection
		if(!$conn){
			echo " Database Error".mysqli_connect_error();
		}
		$id=$_POST["bid"];
	$name=$_POST["bname"];
	$author=$_POST["bauth"];
	$publisher=$_POST["bpub"];
	$price=$_POST["bprice"];
	
}

	</center>
</body>

</html>

File: PythonProject/update.py
#!C:/Python36/python.exe
print("Content-Type: text/html\n\n")
#print("Refresh: 0; url=%s"%"a.html")
print()

import cgi
import pymysql
form=cgi.FieldStorage()
bid=form.getvalue("bid")
'''title=form.getvalue("btitle")
author=form.getvalue("bauthor")
pub=form.getvalue("bpub")'''
print("<html>")
print("<body>")
print("<form action='http://localhost/PythonProject/update2.py'>")
print("<table border='1' align='center'><tr>")
print("<th align='left'>Book id:</th><td><input type='text' value=")
print(bid)
print("name='bid'></td></tr><br>")
print("<th align='left'>Book Title:</th><td><input type='text' name='btitle'></td></tr><br>")
print("<th align='left'>Book Author:</th><td><input type='text' name='bauthor'></td></tr><br>")
print("<th align='left'>Book Publisher:</th><td><input type='text' name='bpub'></td></tr><br>")

print("<tr><td align='center' colspan=2><input type='submit' value='Update'></td></tr>")
print("</table></form></body></html>")
print("<a href='http://localhost/PythonProject/a.html'> Back</a>")
"""con=pymysql.connect('localhost','root','','test')
cur=con.cursor()
#sql="INSERT INTO BOOKS VALUES(%s,%s,%s,%s)",(bid,title,author,pub)
cur.execute("SELECT * FROM BOOKS WHERE Bokid=bid)
result=cur.fetchall()
for row in result:
	bookid=row[0]
	title=row[1]
	author=row[2]
	publisher=row[3]
	
print("Data inserted Successfully")
con.commit()
cur.close()"""